### ISSUE TITLE
Investigate panic in M3 reporter 

### ISSUE THREAD

**Issue by robskillington** [uber-go/tally#45]
```go
panic in handler: send on closed channel
code.uber.internal/recognition/tips/vendor/go.uber.org/fx/modules/yarpc.panicRecovery
	code.uber.internal/recognition/tips/vendor/go.uber.org/fx/modules/yarpc/middleware.go:143
runtime.call32
	/usr/lib/go-1.8/src/runtime/asm_amd64.s:514
runtime.gopanic
	/usr/lib/go-1.8/src/runtime/panic.go:489
runtime.chansend
	/usr/lib/go-1.8/src/runtime/chan.go:180
runtime.selectnbsend
	/usr/lib/go-1.8/src/runtime/chan.go:604
code.uber.internal/recognition/tips/vendor/github.com/uber-go/tally/m3.(*reporter).reportCopyMetric
	code.uber.internal/recognition/tips/vendor/github.com/uber-go/tally/m3/reporter.go:431
code.uber.internal/recognition/tips/vendor/github.com/uber-go/tally/m3.cachedMetric.ReportTimer
	code.uber.internal/recognition/tips/vendor/github.com/uber-go/tally/m3/reporter.go:543
code.uber.internal/recognition/tips/vendor/github.com/uber-go/tally/m3.(*cachedMetric).ReportTimer
	<autogenerated>:11
code.uber.internal/recognition/tips/vendor/github.com/uber-go/tally.(*timer).Record
	code.uber.internal/recognition/tips/vendor/github.com/uber-go/tally/stats.go:177
code.uber.internal/recognition/tips/vendor/github.com/uber-go/tally.(*timer).RecordStopwatch
	code.uber.internal/recognition/tips/vendor/github.com/uber-go/tally/stats.go:189
code.uber.internal/recognition/tips/vendor/github.com/uber-go/tally.Stopwatch.Stop
	code.uber.internal/recognition/tips/vendor/github.com/uber-go/tally/types.go:114
code.uber.internal/recognition/tips/vendor/code.uber.internal/recognition/glue.git/decorators.Metrics.func1.1
	code.uber.internal/recognition/tips/vendor/code.uber.internal/recognition/glue.git/decorators/metrics.go:29
code.uber.internal/recognition/tips/vendor/code.uber.internal/recognition/glue.git/decorators.YARPCScope.func1.1
	code.uber.internal/recognition/tips/vendor/code.uber.internal/recognition/glue.git/decorators/scope.go:36
code.uber.internal/recognition/tips/.gen/glue/handlers/tips.(*wrapper).CreateTip
	code.uber.internal/recognition/tips/.gen/glue/handlers/tips/tips.go:70
code.uber.internal/recognition/tips/.gen/go/recognition/tips/tips/tipsserver.handler.CreateTip
	code.uber.internal/recognition/tips/.gen/go/recognition/tips/tips/tipsserver/server.go:59
code.uber.internal/recognition/tips/.gen/go/recognition/tips/tips/tipsserver.(handler).CreateTip-fm
	code.uber.internal/recognition/tips/.gen/go/recognition/tips/tips/tipsserver/server.go:38
code.uber.internal/recognition/tips/vendor/go.uber.org/yarpc/encoding/thrift.thriftUnaryHandler.Handle
	code.uber.internal/recognition/tips/vendor/go.uber.org/yarpc/encoding/thrift/inbound.go:85
code.uber.internal/recognition/tips/vendor/go.uber.org/yarpc/encoding/thrift.(*thriftUnaryHandler).Handle
	<autogenerated>:8
code.uber.internal/recognition/tips/vendor/go.uber.org/yarpc/internal/inboundmiddleware.unaryChainExec.Handle
	code.uber.internal/recognition/tips/vendor/go.uber.org/yarpc/internal/inboundmiddleware/chain.go:72
code.uber.internal/recognition/tips/vendor/go.uber.org/yarpc/internal/inboundmiddleware.(*unaryChainExec).Handle
	<autogenerated>:2
code.uber.internal/recognition/tips/vendor/go.uber.org/fx/modules/yarpc.authInboundMiddleware.Handle
	code.uber.internal/recognition/tips/vendor/go.uber.org/fx/modules/yarpc/middleware.go:81
code.uber.internal/recognition/tips/vendor/go.uber.org/fx/modules/yarpc.(*authInboundMiddleware).Handle
	<autogenerated>:15
code.uber.internal/recognition/tips/vendor/go.uber.org/yarpc/internal/inboundmiddleware.unaryChainExec.Handle
	code.uber.internal/recognition/tips/vendor/go.uber.org/yarpc/internal/inboundmiddleware/chain.go:76
code.uber.internal/recognition/tips/vendor/go.uber.org/yarpc/internal/inboundmiddleware.(*unaryChainExec).Handle
	<autogenerated>:2
code.uber.internal/recognition/tips/vendor/go.uber.org/fx/modules/yarpc.panicInboundMiddleware.Handle
	code.uber.internal/recognition/tips/vendor/go.uber.org/fx/modules/yarpc/middleware.go:123
code.uber.internal/recognition/tips/vendor/go.uber.org/yarpc/internal/inboundmiddleware.unaryChainExec.Handle
	code.uber.internal/recognition/tips/vendor/go.uber.org/yarpc/internal/inboundmiddleware/chain.go:76
code.uber.internal/recognition/tips/vendor/go.uber.org/yarpc/internal/inboundmiddleware.(*unaryChainExec).Handle
	<autogenerated>:2
code.uber.internal/recognition/tips/vendor/go.uber.org/fx/modules/yarpc.contextInboundMiddleware.Handle
	code.uber.internal/recognition/tips/vendor/go.uber.org/fx/modules/yarpc/middleware.go:53
code.uber.internal/recognition/tips/vendor/go.uber.org/yarpc/internal/inboundmiddleware.unaryChainExec.Handle
	code.uber.internal/recognition/tips/vendor/go.uber.org/yarpc/internal/inboundmiddleware/chain.go:76
code.uber.internal/recognition/tips/vendor/go.uber.org/yarpc/internal/inboundmiddleware.(*unaryChainExec).Handle
	<autogenerated>:2
code.uber.internal/recognition/tips/vendor/go.uber.org/yarpc/internal/observability.(*Middleware).Handle
	code.uber.internal/recognition/tips/vendor/go.uber.org/yarpc/internal/observability/middleware.go:75
code.uber.internal/recognition/tips/vendor/go.uber.org/yarpc/internal/inboundmiddleware.unaryChainExec.Handle
	code.uber.internal/recognition/tips/vendor/go.uber.org/yarpc/internal/inboundmiddleware/chain.go:76
```

**Comment by robskillington** [comment]
cc @mcgourty

**Comment by robskillington** [comment]
Fixed with #46 and released with https://github.com/uber-go/tally/releases/tag/v3.0.2
